app/src/main/java/com/android/voyageur/ui/profile/ProfileScreen.kt

```diff
@@ -149,84 +167,61 @@ fun ProfileContent(
  *   accepting or rejecting friend requests.
  */
 @Composable
-fun ExpandableFriendReqMenu(
+fun FriendReqMenu(
     friendRequests: List<FriendRequest>,
     notificationUsers: List<User>,
     userViewModel: UserViewModel,
 ) {
-  var expanded by remember { mutableStateOf(false) } // Toggle for expanded state
-  // Collapsed Menu
+  // Parent Card containing the scrollable box
   Card(
       shape = RoundedCornerShape(12.dp),
-      modifier = Modifier.fillMaxWidth(0.80f).padding(horizontal = 10.dp, vertical = 8.dp),
-  ) {
-    Column(modifier = Modifier.padding(12.dp)) {
-      // Header row with title and expand/collapse button
-      Row(
-          modifier = Modifier.fillMaxWidth(),
-          verticalAlignment = Alignment.CenterVertically,
-          horizontalArrangement = Arrangement.SpaceBetween) {
-            Text(
-                text = "${friendRequests.size} Pending Friend Requests",
-                color = MaterialTheme.colorScheme.onSurface)
-
-            IconButton(onClick = { expanded = true }) {
-              Icon(
-                  imageVector = Icons.Default.KeyboardArrowDown,
-                  contentDescription = "Expand",
-                  tint = MaterialTheme.colorScheme.onSurface)
-            }
-          }
-    }
-  }
+      modifier =
+          Modifier.fillMaxWidth(0.80f)
+              .padding(horizontal = 10.dp, vertical = 8.dp)
+              .testTag("friendRequestCard")) {
+        Column(modifier = Modifier.padding(12.dp)) {
+          Text(
+              text = stringResource(R.string.pending_friend_requests, friendRequests.size),
```

Comment:
I think we should avoid using formatted strings as string resources. I understand the idea of not hard coding strings, but having a format like this is not always beneficial, I believe that the version before was a bit more readable.

 But, maybe if we decide to keep it like we can add multiple language support in the future, if we run out of features to code :)

Reply:
I'll keep this in mind for the future!